assets/i18n/es.po

```diff
@@ -0,0 +1,720 @@
+
+#: assets/page/SettingsAdminUsers.svelte:111
+msgid "Delete user"
+msgstr "Eliminar usuario"
+
+#: assets/page/Help.svelte:79
+msgid "Demo error pages"
+msgstr "Páginas de error de demostración"
+
+#: assets/page/SettingsAdmin.svelte:127
+msgid "Disk usage"
+msgstr "Uso de disco"
+
+#: assets/page/Chat.svelte:230
+msgid "Do you want to chat with \"%1\"?"
+msgstr "¿Quieres conversar con \"%1\"?"
+
+#: assets/page/Chat.svelte:223
+msgid "Do you want to create the connection \"%1\"?"
+msgstr "¿Deseas crear la conexión \"%1\"?"
+
+#: assets/page/Fallback.svelte:28
+msgid "Downloaded Convos."
+msgstr "Convos descargado."
+
+#: assets/page/Login.svelte:102
+msgid "E-mail"
+msgstr "Correo electrónico"
+
+#: assets/page/SettingsAccount.svelte:72
+msgid "Email"
+msgstr "Correo electrónico"
+
+#: assets/page/SettingsAccount.svelte:100
+msgid "Enable compact message display"
+msgstr "Activar vista compacta de mensajes"
+
+#: assets/page/SettingsAccount.svelte:80
+msgid "Enable notifications"
+msgstr "Activar notificaciones"
+
+#: assets/page/ConversationAdd.svelte:56
+msgid "Enter the name of an exising conversation, or create a new conversation."
+msgstr "Ingresa el nombre de una conversación existente, o crea una nueva conversación."
+
+#: assets/components/ConnectionForm.svelte:135
+msgid "Ex: #convos"
+msgstr "Ej: #convos"
+
+#: assets/components/ConnectionForm.svelte:125
+msgid "Ex: Clark Kent"
+msgstr "Ej: Juan Pérez"
+
+#: assets/components/ConnectionForm.svelte:114
+msgid "Ex: chat.freenode.net:6697"
+msgstr "Ej: chat.freenode.net:6697"
+
+#: assets/page/SettingsAdmin.svelte:109
+msgid "Ex: https://meet.jit.si/"
+msgstr "Ej: https://meet.jit.si/"
+
+#: assets/page/SettingsAdmin.svelte:86
+msgid "Ex: jhthorsen@cpan.org"
+msgstr "Ej: jhthorsen@cpan.org"
+
+#: assets/page/Login.svelte:101
+msgid "Ex: john@doe.com"
+msgstr "Ej: juan@perez.com"
+
+#: assets/components/ConnectionForm.svelte:121
+msgid "Ex: superman"
+msgstr "Ej: superman"
+
+#: assets/page/SettingsAccount.svelte:88
+msgid "Expand URL to media"
+msgstr "Expandir URL a multimedia"
+
+#: assets/page/SettingsAdmin.svelte:100
+msgid "Force default connection"
+msgstr "Forzar conexión por defecto"
+
+#: assets/page/SettingsAdminUsers.svelte:135
+msgid "Generate link"
+msgstr "Generar enlace"
+
+#: assets/page/SettingsAdmin.svelte:68
+msgid "Global settings"
+msgstr "Ajustes globales"
+
+#: assets/page/Fallback.svelte:41
+msgid "Go to start page"
+msgstr "Ir a la página de inicio"
+
+#: assets/components/ChatSidebar.svelte:197
+msgid "Help"
+msgstr "Ayuda"
+
+#: assets/page/Login.svelte:78
+msgid "Hint: Use a phrase from a book."
+msgstr "Pista: Usa una frase de un libro."
+
+#: assets/components/ConnectionForm.svelte:115
+msgid "Host and port"
+msgstr "Servidor y puerto"
+
+#: assets/page/SettingsAccount.svelte:84
+msgid "Ignore join/part messages"
+msgstr "Ignorar mensajes de entra/sale"
+
+#: assets/page/SettingsAdmin.svelte:131
+msgid "Inode usage"
+msgstr "Uso de ínodo"
```

Comment:
I get the impression "inodo" (with the stress in the second syllable) is the more commonly accepted Spanish version of this word. It definitely sounds like that based on eg. [Google results](https://www.google.com/search?hl=en&q=%C3%ADnodo).